aries-site/src/examples/templates/dashboards/components/UpdatesFeed.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { Box, Card, CardBody, CardHeader, ThemeContext } from 'grommet';
+import { useContext } from 'react';
+import { DashboardCardHeader } from '.';
+
+export const UpdatesFeed = ({ background, children, menuItems, title }) => {
+  const theme = useContext(ThemeContext);
+  const { body, header } = theme.card;
+
+  return (
+    <Card>
+      <Box background={background || 'background-front'}>
```

Comment:
Is the `|| background-front` part necessary?

```suggestion
      <Box background={background}>
```